deis-dev/tpl/deis-objectstorage-secret.yaml

```diff
@@ -8,7 +8,9 @@ metadata:
   annotations:
     deis.io/objectstorage: "{{.storage}}"
 type: Opaque
-data: {{ if eq .storage "gcs" }}
+data: {{ if eq .storage "minio" }}
+  access-key-id: OFRaUlkySlJXTVBUNlVNWFI2STUK
```

Comment:
Right now minio-user is also hardcoded. We can change this from hard-coding to generating some random key and secret once all the repo uses the object-store secret instead of minio-user